Update README.md

martinschwinzerl opened the pull request:
Updated the README.md file to properly present the most recent build instructions, including the officially supported way to build and install the python bindings.

martinschwinzerl committed:
Update README.md

Updated the README.md file to properly present the most recent build instructions, including the officially supported way to build and install the python bindings.

rdemaria commented on README.md:
```diff
@@ -8,32 +8,59 @@
 - _optionally_ googletest library for running the unit-tests
 - _optionally_ NVidia CUDA 7.5 or higher 
 - _optionally_ OpenCL 1.2 or higher (with C++ host bindings)
+- _optionally_ Python 3.5 or higher with numpy, scipy, cobjects, pysixtrack, and sixtracktools modules (for Python bindings)
```
Python 3.7 indeed on pysixtrack...

rdemaria commented on README.md:
````diff
@@ -8,32 +8,59 @@
 cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=~/opt/sixtracklib
 ```
-If you have enabled the building of unit tests in the settings file, you may have to pass the location of the googletest library location to cmake. Assuming you installed googletest with the install prefix `/opt/googletest`, please set the `SIXTRACKL_GOOGLETEST_ROOT` variable in `Settings.cmake` to this value.
+If you have enabled the building of unit tests in the settings file, the build configuration will attempt to find a recent ersion of the googletest library on your system and, failing that, will try to download the latest version from github and build a version you may have to pass the location of the googletest library location to cmake. Assuming you installed googletest with the install prefix  ...
````
type: ersion -> version

rdemaria commented on README.md:
````diff
@@ -8,32 +8,59 @@
 make install
 ```
+### Installing SixTrackLib Python bindings
+If you had enabled the python bindings during the configuration step and have successfully completed the creation of the library itself, please proceed as follows to install the Python bindings:
````
typo: python -> Python

martinschwinzerl committed:
Update README.md

Thank you for the review and the update & sorry for the glitches!
Changes:
- Incremented required Python version to 3.7
- Fixed typo ersion -> version
- Always use upper-case Python if it is not in reference to a positively lower-case path

Additionally: 
- Fixed a missing "to"
- Rephrased and hopefully clarified the section about build-configuration using cmake command line parameters

Thank you!
